src/celeritas/ext/detail/CelerEmStandardPhysics.cc

```diff
@@ -0,0 +1,396 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020-2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/detail/CelerEmStandardPhysics.cc
```

Comment:
I'm not sure how to make git recognize this is a rename of GeantPhysicsList.cc (with only a couple small changes).

Reply:
Git's a bit tricky about that (and I don't think there's fundamentally any metadata that says it's that way). The new class name is probably also messing up the diff. I wouldn't worry about it; I think it'll be obvious enough when viewing the history... the only thing to do would be to have separate commits for "move file", "find-replace name", and "actually change stuff".... but that would only really show up in the review rather than the final squash.

Reply:
It seems to have figured it out when I renamed the old physics list.